tests/security/389ds/tls_389ds_server.pm

```diff
@@ -107,6 +108,22 @@ sub run {
     # Set the ldap_uri with LDAP over SSL (LDAPS) Certificate
     assert_script_run("sed -i 's/^ldap_uri =.*\$/ldap_uri = ldaps:\\/\\/$local_name.example.com/' /tmp/sssd.conf");
 
+    # Permit ssh/scp from client as root
+    # Due to bsc#1173067, openssh now no longer allows RootLogin
+    # using password auth on Tumbleweed, the latest Leap 16 will
+    # sync with Tumbleweed as well
+    if (is_sle or is_leap('<16.0')) {
```

Comment:
leap and sle will always be in sync here (they use the same binaries after all)
so sle 16 for sure also will  have that (if not, I'd consider it a sec issue on sLE16; so making sure this really makes it in SLE 16 is probably best right now)

Reply:
Yup, thanks for the kindly update! Let me revise the code

Reply:
done, please check again.

Reply:
done, please check again.

> Please, a reminder for everybody!
> 
> When you clone a job as a verification run, ensure it is NOT added to group/1, as this falsifies the information seen about the snapshot.
> 
> If you use openqa-clone-custom-git-refspec this is happening automatically for you.
Thanks for the kindly reminder, will keep it in mind.